apps/frontend/src/network/users/api.ts

```diff
@@ -30,6 +31,23 @@ export const getUser = async (
   return resp.json();
 };
 
+export const getUsersWithAlgolia = async (
```

Comment:
can we call this getUsers and the legacy function getUsersLegacy ?

Reply:
Yes I will change it 👍